recipes/orange3/meta.yaml

```diff
@@ -0,0 +1,63 @@
+package:
+  name: orange3
+  version: 3.3.6
+
+source:
+  fn: Orange3-3.3.6.tar.gz
+  url: https://github.com/biolab/orange3/archive/3.3.6.tar.gz
+  md5: 927f3702ea5c9f1c8d1b59478476b4d5
+
+build:
+  number: 0
+  script:
+    - python setup.py install
+  entry_points:
+    - orange-canvas = Orange.canvas.__main__:main
+  osx_is_app: True
+  skip: True  # [py2k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - scipy
+  run:
+    - python
+    - setuptools
+    - scipy
+    - scikit-learn
+    - bottlechest
+    - chardet
+    - docutils
+    - joblib
+    - xlrd
+    - pyqt
+    - pyqtgraph
+    - icu         # [linux]
```

Comment:
I believe this is not "really" needed, but you had to add it b/c of an inconsistency in `conda-forge`'s vs `defaults` stack. Right? (See https://github.com/conda-forge/matplotlib-feedstock/issues/36)

If that is the case can you please add a comment above that this is a temporary workaround and we should re-visit once the problem is fixed. (Just to avoid over-specifying dependencies.)